client/__tests__/e2e/e2e.test.ts

```diff
@@ -55,14 +55,14 @@ import {
 
 import { datasets } from "./data";
 
-import { scaleMax } from "../../src/util/camera";
 import {
   DATASET,
   pageURLTruncate,
   testURL,
   pageURLSpatial,
 } from "../common/constants";
 import { goToPage } from "../util/helpers";
+import { SCALE_MAX } from "../../src/util/constants";
```

Comment:
import constant from its own file to prevent importing openseadragon when running e2e tests `npm run e2e`